tests/nested/manual/split-refresh/task.yaml

```diff
@@ -0,0 +1,224 @@
+summary: Check app refreshes don't wait for a reboot in hybrid systems
+
+details: |
+  Check that (on hybrid systems) refreshes of both essential (e.g., kernel,
+  gadget) and non-essential snaps don't make apps and non-model bases wait for
+  a reboot like the essential snaps. Check that if an app depends on the model
+  base, it does wait for its refresh, and consequently, a reboot.
+
+systems: [ubuntu-22.04-64, ubuntu-24.04-64]
+
+environment:
+    NESTED_ENABLE_TPM: false
+    NESTED_ENABLE_SECURE_BOOT: false
+    NESTED_BUILD_SNAPD_FROM_CURRENT: true
+    NESTED_ENABLE_OVMF: true
+    NESTED_REPACK_KERNEL_SNAP: true
+    # store related setup
+    STORE_ADDR: localhost:11028
+    STORE_DIR: $(pwd)/fake-store-blobdir
+
+prepare: |
+  if [ "$TRUST_TEST_KEYS" = "false" ]; then
+      echo "This test needs test keys to be trusted"
+      exit
+  fi
+  echo "Install used snaps"
+  snap install test-snapd-curl --devmode --edge
+  snap install jq --devmode --edge
+  if [ -d /var/lib/snapd/seed ]; then
+      mv /var/lib/snapd/seed /var/lib/snapd/seed.orig
```

Comment:
Could this be:
```sh
tests.backup prepare /var/lib/snapd/seed
tests.cleanup defer tests.backup restore /var/lib/snapd/seed
```

Reply:
I've made this change now. Thank you

Reply:
I restored this to its previous state because the restore function fails if the path doesn't exist so it becomes more complicated to deal with given the file/directory distinction